tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -9,6 +9,18 @@
 
 class DocumentationCommentTest(unittest.TestCase):
 
+    Description = DocumentationComment.Description
+    Parameter = DocumentationComment.Parameter
+    ReturnValue = DocumentationComment.ReturnValue
+
+    def load_testdata(self, filename):
+        filename = (os.path.dirname(os.path.realpath(__file__)) +
+                    "/documentation_extraction_testdata/" + filename)
+        with open(filename, "r") as fl:
```

Comment:
@SanketDG In some fonts, the **l** (el) are indistinguishable from the numerals one. Maybe we avoid using them? Maybe it can be made better as:
```python
test_filename = (...)
with open(test_filename,'r') as test_file:
    data = test_file.read()
```